api/core/number.go

```diff
@@ -94,14 +94,14 @@ func (n Number) AsUint64() uint64 {
 
 // - as x atomic
 
-// AsNumberAtomic gets the raw, uninterpreted raw value. Might be useful for
-// some atomic operations.
+// AsNumberAtomic atomically gets the raw, uninterpreted raw
+// value. Might be useful for some atomic operations.
 func (n *Number) AsNumberAtomic() Number {
 	return NewNumberFromRaw(n.AsRawAtomic())
 }
 
-// AsRawAtomic gets atomically the raw, uninterpreted raw value. Might
-// be useful for some atomic operations.
+// AsRawAtomic atomically gets atomically the raw, uninterpreted raw
+// value. Might be useful for some atomic operations.
```

Comment:
Similarly, 'AsNumberAtomic gets uninterpreted raw value atomically. Might be useful for some atomic operations'

Reply:
Thanks, fixed.